<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordSetDialog</class>
 <widget class="QDialog" name="PasswordSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="CommonPushButton" name="pushButtonPwdSetBack">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>190</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退格</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetNum3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>220</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetNum9">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>220</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetNum6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>290</y>
     <width>220</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetNum8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>220</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetNum5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>220</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetNum2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>220</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetNum7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>220</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetNum1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>220</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetNum4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>220</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetClear">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>290</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetNum0">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>220</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QFrame" name="framePasswordSetting">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>30</y>
     <width>700</width>
     <height>140</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetReturn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>490</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetSave">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>390</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>120</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="FMyLineEdit" name="lineEditCheckPassword">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="FMyLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>50</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>新 密  码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetDot">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>220</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonPwdSetShop">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>490</y>
     <width>220</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>#</string>
   </property>
  </widget>
  <zorder>framePasswordSetting</zorder>
  <zorder>pushButtonPwdSetBack</zorder>
  <zorder>pushButtonPwdSetNum3</zorder>
  <zorder>pushButtonPwdSetNum9</zorder>
  <zorder>pushButtonPwdSetNum6</zorder>
  <zorder>pushButtonPwdSetNum8</zorder>
  <zorder>pushButtonPwdSetNum5</zorder>
  <zorder>pushButtonPwdSetNum2</zorder>
  <zorder>pushButtonPwdSetNum7</zorder>
  <zorder>pushButtonPwdSetNum1</zorder>
  <zorder>pushButtonPwdSetNum4</zorder>
  <zorder>pushButtonPwdSetClear</zorder>
  <zorder>pushButtonPwdSetNum0</zorder>
  <zorder>pushButtonPwdSetReturn</zorder>
  <zorder>pushButtonPwdSetSave</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEditCheckPassword</zorder>
  <zorder>lineEditPassword</zorder>
  <zorder>label</zorder>
  <zorder>pushButtonPwdSetDot</zorder>
  <zorder>pushButtonPwdSetShop</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>FMyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>fmylineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>lineEditCheckPassword</tabstop>
  <tabstop>pushButtonPwdSetNum1</tabstop>
  <tabstop>pushButtonPwdSetNum2</tabstop>
  <tabstop>pushButtonPwdSetNum3</tabstop>
  <tabstop>pushButtonPwdSetNum4</tabstop>
  <tabstop>pushButtonPwdSetNum5</tabstop>
  <tabstop>pushButtonPwdSetBack</tabstop>
  <tabstop>pushButtonPwdSetNum6</tabstop>
  <tabstop>pushButtonPwdSetNum7</tabstop>
  <tabstop>pushButtonPwdSetNum8</tabstop>
  <tabstop>pushButtonPwdSetNum9</tabstop>
  <tabstop>pushButtonPwdSetNum0</tabstop>
  <tabstop>pushButtonPwdSetClear</tabstop>
  <tabstop>pushButtonPwdSetSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
